src/GovUk.Education.ExploreEducationStatistics.Content.Api.Tests/Services/ReleaseFileServiceTests.cs

```diff
@@ -17,14 +22,23 @@
 using Xunit;
 using static GovUk.Education.ExploreEducationStatistics.Common.BlobContainers;
 using static GovUk.Education.ExploreEducationStatistics.Common.Model.FileType;
-using static GovUk.Education.ExploreEducationStatistics.Common.Services.FileStorageUtils;
+using static GovUk.Education.ExploreEducationStatistics.Common.Services.CollectionUtils;
 using static GovUk.Education.ExploreEducationStatistics.Content.Model.Database.ContentDbUtils;
-using File = GovUk.Education.ExploreEducationStatistics.Content.Model.File;
+using File = System.IO.File;
 
 namespace GovUk.Education.ExploreEducationStatistics.Content.Api.Tests.Services
 {
-    public class ReleaseFileServiceTests
+    public class ReleaseFileServiceTests : IDisposable
     {
+        private readonly List<string> _filePaths = new();
+
+        public void Dispose()
+        {
+            // Cleanup any files that have been
+            // written to the filesystem.
+            _filePaths.ForEach(File.Delete);
+        }
+
         [Fact]
         public async Task Stream()
         {
```

Comment:
Think this test is now `StreamFile` not `Stream`

Reply:
Done!